src/main/java/Calculator.java

```diff
@@ -0,0 +1,80 @@
+import java.util.Collections;
+import java.util.LinkedList;
+import java.util.Objects;
+
+public class Calculator {
+    private final LinkedList<String> inputValue;
+
+    private IntConverter intConverter;
+
+    private int answer;
+
+    public Calculator(LinkedList<String> queue) {
+        this.inputValue = queue;
+        this.intConverter = new IntConverter();
+        this.answer = 0;
+    }
+
+    public static Calculator of(String inputVal) {
+        isValid(inputVal);
+        String[] result = inputVal.split(" ");
```

Comment:
```suggestion
        String[] result = inputVal.split(DELIMITER);
```
지금은 공백이지만 나중엔 아닐 수도 있겠죠?

Reply:
@catsbi 피드백 감사합니다 :) 역시 따로 선언해 분리했습니다.